src/autoscaler/cf/oauth.go

```diff
@@ -20,6 +20,9 @@ var (
 )
 
 func (c *cfClient) IsUserSpaceDeveloper(userToken string, appId string) (bool, error) {
+	if !c.isValidUserToken(userToken) {
```

Comment:
How about using "isValidUserToken()" directly in upper level to skip` IsUserSpaceDeveloper() ` and `IsUserAdmin()` directly with an invalid token?